Dockerfile.ubi

```diff
@@ -279,22 +301,24 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 FROM nvcr.io/nvidia/cuda:${CUDA_MAJOR}.${CUDA_MINOR}.${CUDA_PATCH}-devel-ubi9 AS runtime
 
 ARG CUDA_MAJOR=12
-ARG CUDA_MINOR=8
+ARG CUDA_MINOR=9
 ARG CUDA_PATCH=1
 ARG PYTHON_VERSION
 ARG NVSHMEM_VERSION=3.3.20
+ARG PYTORCH_CUDA_INDEX_BASE_URL=https://download.pytorch.org/whl
 
-
-
-RUN cat > /tmp/constraints.txt << 'EOF'
+RUN cat > /tmp/constraints.txt <<'EOF'
 torch==2.7.1
 EOF
 
 ENV LANG=C.UTF-8 \
     LC_ALL=C.UTF-8 \
     PYTHON_VERSION=${PYTHON_VERSION:-3.12} \
-    UV_TORCH_BACKEND=${UV_TORCH_BACKEND:-cu${CUDA_MAJOR}${CUDA_MINOR}} \
+    # UV_TORCH_BACKEND=${UV_TORCH_BACKEND:-cu${CUDA_MAJOR}${CUDA_MINOR}} \
```

Comment:
is this safe from an ABI perspective? Should we attempt to build with torch 2.8 even though that means we can't use precompiled vllm wheels?